public/gamecompare.php

```diff
@@ -91,6 +82,7 @@
             echo GetGameAndTooltipDiv($gameID, $gameTitle, $gameIcon, $consoleName, false, 96);
 
             echo "<form method=get action='/gamecompare.php'>";
+            echo csrf_field();
```

Comment:
Good catch, the csrf field, while not doing any harm in a GET from, is not required here. I went through all 74 occurrences of `<form>` to see if the csrf field should be applied to a POST form. I did not catch that one because it's missing the double quotes... I removed `method="get"` as that's the default method of forms.

Reply:
>  while not doing any harm in a GET from
It breaks bookmarks and the ability to share a URL with another user. This is one page that we sometimes link for moderation (though usually just the landing page, not the reload after changing the selected game), so the ability to share the URL is fairly important.